install_and_cache_pkgs.sh

```diff
@@ -18,28 +18,6 @@ cache_dir="${1}"
 # List of the packages to use.
 input_packages="${@:3}"
 
-# Trim commas, excess spaces, sort, and version syntax.
```

Comment:
Can we retain the comments here?

Reply:
The reason I removed the comments they were incorrect and the function defines what it does.

The impact of this is you either get code churn when what it does changes or even worse incorrect comments when the function changes but all uses aren't update.

This is what happened when the the syntax was changed from colon to equals, making this comment incorrect.

Thoughts?